tests/layer_tests/pytorch_tests/test_quantized_relu6.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2018-2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import platform
+import numpy as np
+import pytest
+import torch
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+
+class quantized_relu6(torch.nn.Module):
+    def __init__(self, scale, zero_point, dtype) -> None:
+        torch.nn.Module.__init__(self)
+        self.scale = scale
+        self.zero_point = zero_point
+        self.dtype = dtype
+
+    def forward(self, input_tensor):
+        quantized_tensor = torch.quantize_per_tensor(input_tensor, 1.0, 0, self.dtype)
+        q_relu6 = torch.ops.quantized.relu6(quantized_tensor, self.scale, self.zero_point)
```

Comment:
```suggestion
        q_relu6 = torch.ops.quantized.relu6(quantized_tensor, self.scale, self.zero_point)
```

`torch.ops.quantized.relu6` have different signature. It accepts tensor and inplace flag.

Reply:
thank you for pointing it out, I have modified the pytest file as well as the cpp file to accept those 2 inputs only

all tests pass successfully now